modules/DMZ/models/classification/digit_recognition

```diff
@@ -0,0 +1,99 @@
+#Digit Recognition Module
+
+# Standard scientific Python imports
+import matplotlib
+matplotlib.use('Agg')   #Force matplotlib to not use any Xwindows backend (causes errors)
+from sklearn import svm # Import svm classifier
+import skimage.io       # reading images
+import os     # directories
+import imghdr # image file recognition
+import numpy  # numpy arrays
+
+class digit_recognition:
+    train = []
+    test  = []
+
+    # the init function will enter the paths given, which contain the images needed for testing and training
+    # init will put them into the format needed for the classifier, and return a tuple of two lists
+    # corresponding to the testing and training data
+    # returns a zipped list, where element[0] is the testing data and [1] is the training data
+    # unzip to get both the images and the predictions
+    def __init__(self, path):
```

Comment:
You do not want all of this to happen immediately upon creation of the class. You want it to only happen when the framework wants to run this algorithm against images. Please create several methods that break up the necessary tasks. Also, if any methods seem like they could be useful in the rest of the framework, please break them out into a new function or file in the utility folder in DMZ. That way the rest of the project has a standard way of accessing them.